src/community/jdbcconfig/pom.xml

```diff
@@ -52,7 +52,7 @@
       <!-- blazing fast LZW compression/decompression -->
       <groupId>com.ning</groupId>
       <artifactId>compress-lzf</artifactId>
-      <version>0.9.1</version>
+      <version>1.0.3</version>
```

Comment:
I am used to this being handled in root pom.xml dependency management section (so we do not get conflicts between community modules).

Is there a reason why this is hard coded ?